content/docs/sidebar.json

```diff
@@ -53,10 +53,13 @@
         }
       },
       {
-        "slug": "experiments",
+        "slug": "ml-pipelines",
```

Comment:
Just looking at the slugs and titles. How do we differentiate "Data Pipelines" from "ML Pipelines"? ML is data driven so it at first glance it just looks like maybe it'll be a more specific version of the previous chapter.